batcher/aligned/src/main.rs

```diff
@@ -107,20 +109,87 @@ pub struct SubmitArgs {
     keystore_path: Option<PathBuf>,
     #[arg(name = "Private key", long = "private_key")]
     private_key: Option<String>,
-    #[arg(
-        name = "Max Fee (ether)",
-        long = "max_fee",
-        default_value = "0.0013ether" // 13_000 gas per proof * 100 gwei gas price (upper bound)
-    )]
-    max_fee: String, // String because U256 expects hex
+    #[command(flatten)]
+    price_estimate: PriceEstimateArgs,
```

Comment:
commands tend to be verbs when possible for bettery clarity. a `price estimate` is a noun. This is why i had suggested the function to be named something like `suggestGasPrice` . This makes the function easier to understand and easier to follow